ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json

```diff
@@ -245,6 +245,30 @@
           "class": "form-control"
         }
       }
+    },
+    {
+      "preset_name": "date_ab-datepicker",
+      "values": {
+        "form_snippet": "date_ab-datepicker.html",
+        "display_snippet": "date.html",
+        "validators": "scheming_required isodate from_date_is_before_until_date convert_to_json_if_date",
+        "form_attrs": {
+          "data-datepicker": "",
+          "class": "form-control"
+        }
```

Comment:
These are probably not required as it has its own template anyway

Reply:
Resolved in [8910a57](https://github.com/vrk-kpa/opendata/pull/1757/commits/8910a5741af49cdb1b52f82f15fee20e0e92071e)